keyboards/numatreus/keymaps/like_jis/keymap.c

```diff
@@ -0,0 +1,213 @@
+#include QMK_KEYBOARD_H
+#include "keymap_jp.h"
+
+
+#ifdef RGBLIGHT_ENABLE
+//Following line allows macro to read current RGB settings
+extern rgblight_config_t rgblight_config;
+#endif
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+enum layer_number {
+  _BASE = 0,
+  _NUMPAD,
+  _LOWER,
+  _RAISE,
+  _ADJUST,
+};
+
+enum custom_keycodes {
+  BASE = SAFE_RANGE,
+  NUMPAD,
+  KANJI,
+  LOWER,
+  RAISE,
+  ADJUST,
+  RGBRST
+};
+
+enum tapdances{
+  TD_CODO = 0,
+  // TD_MNUB,
+};
+
+// Layer Mode aliases
+#define KC_LOWER LOWER
+#define KC_RAISE RAISE
+#define KC_ADJST ADJUST
+#define KC_DLNP NUMPAD //DF(_NUMPAD)
+#define KC_DLBS BASE //DF(_BASE)
+
+#define KC______ KC_TRNS
+#define KC_XXXXX KC_NO
+#define KC_KANJI KANJI
+
+#define KC_RST   RESET
+#define KC_LRST  RGBRST
+#define KC_LTOG  RGB_TOG
+#define KC_LHUI  RGB_HUI
+#define KC_LHUD  RGB_HUD
+#define KC_LSAI  RGB_SAI
+#define KC_LSAD  RGB_SAD
+#define KC_LVAI  RGB_VAI
+#define KC_LVAD  RGB_VAD
+#define KC_LMOD  RGB_MOD
+#define KC_KNRM  AG_NORM
+#define KC_KSWP  AG_SWAP
```

Comment:
These can be placed directly into the keymap.

Reply:
It has changed.